### DISCUSSION TITLE
Golbal CSS problem in next js

### DISCUSSION THREAD

**Discussion by kryptoDev0777** [community/community#38333] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, everyone.
I am creating new project in next.js
but I don't know how to use global CSS in next js.
I copied and pasted on some style codes on global style, 
then it works, but project loading speed is very slow now, I can't click even any button.
Who can help me?

**Comment by DSDmark** [comment, marked as answer]
Dude... really believe me, we are in the same boat.
Web framework is really suck when come to CSS. Only way you can also this problem why learn how to use component base style and also when you use web framework forget CSS even exists in real life. Instead, use CSS framework. It's not good choice to style from scratch if you use web-framework, then use CSS library or framework for better control in your component. It's help to scale your project in future.

For some better explanation, you can see that video:-
[**7 way to apply CSS**](https://youtu.be/ouncVBiye_M)

**Reply by DSDmark** [reply]
waythewhy I recommend you to start if you made up your mind with CSS framework then start first **boosterp 5**

**Reply by kryptoDev0777** [reply]
Thanks for your reply
I am interesting your answer..
could you let me know how to use CSS framework in my case?

**Reply by DSDmark** [reply]
Hi @mgx5kw,

you mention that you're ``` project loading speed is very slow now```.
Is that because of **CSS** or you use multiple packages.
In the case of packages, I think you use, by default, **webpack**. When in the time of production, it's make bundle size very large. 
You could try this [**Parcel**](https://www.npmjs.com/package/parcel)

**Reply by DSDmark** [reply]
And for **CSS** things if speed all you need then use [**Tailwind**](https://tailwindcss.com/) there is no one compare to fast like **tailwinds** now day's it's best

**Reply by kryptoDev0777** [reply]
project is large, but its speed was not bad before I add style

**Reply by DSDmark** [reply]
So use tailwind, go to its official size and read docs. It's make minimum file size and remove unnecessary CSS when you build your project

**Reply by kryptoDev0777** [reply]
I solved problem myself.

**Comment by jfullstackdev** [comment]
I'm not very sure of that framework but frameworks are really needed particularly in mobile-first development

maybe what's happening is that, you installed so many packages that are not needed, you just want to install what you need or else that will really slow you down, even in Java, that is the main reason

and there are lightweight frameworks like this if you into mobile-first CSS framework, depending on the size of your project

https://www.w3schools.com/w3css/defaulT.asp
